vnext/ReactCopies/RNTester/js/components/TextInlineView.js

```diff
@@ -25,6 +25,21 @@ function Basic(): React.Node {
   );
 }
 
+function NestedTexts(): React.Node {
+  return (
+    <View>
+      <Text>This is the first row</Text>
+      <Text>
+        <Text>
+          <Text>This is a nested text </Text>
+          <View style={{height: 20, width: 20, backgroundColor: 'red'}} />
```

Comment:
I'm assuming this doesn't work for windows right now?

Reply:
Yeah, this was only added to the Android TextExample, so we don't ever see this added to our tester, but IIRC we YellowBox on nested text elements. I think we have an issue somewhere about nested text but am having trouble finding it.